test/scenarios/17. settings/advanced/show-test-networks.csv

```diff
@@ -0,0 +1,12 @@
+Advanced settings: Check “Show test networks” toggle works correctly,,,,
+Step,Test Steps,Test Data,Expected Result,Notes
+1,Open the extension.,,The Welcome Back screen is shown,
+2,Proceed to Unlock the wallet.,password (8 characters min),,
+3,Click on the right corner setting(three dots) and click on the Settings from menu.,,Settings page appears,
+4,Click on the Advanced tab.,,Advanced settings page appears,
+5,Validate that the 'Show test networks' is default to off.,,The toggle is off by default,
```

Comment:
these two sentences are the same to me, `default to off` = `is off by default,` wdyt?

Reply:
Actual this format is difficult to read. So it's actual test step and the expected test result which are the same.

<img width="1472" alt="Screenshot 2024-02-23 at 4 05 35 PM" src="https://github.com/MetaMask/metamask-extension/assets/153644847/e64be248-1d62-4596-94f0-d42b01c0ab6a">

Reply:
I could make it different to read.